src/js/Draw/L.PM.Draw.Marker.js

```diff
@@ -194,4 +196,62 @@ Draw.Marker = Draw.extend({
       this._hintMarker?.setIcon(this.options.markerStyle.icon);
     }
   },
+  _setupTouchScreenTips() {
+    let markerHintTip = null;
+
+    this._map.on('pm:drawstart', (event) => {
+      if (event.shape === 'Marker' && !deviceHasFinePointer()) {
+        // Hides the "hint marker" as that is confusing for touch screen users.
+        const drawMarker = this._map.pm.Draw.Marker;
+        drawMarker._hintMarker.remove();
+
+        // Creates the touch screen hint "permanently" (while using the tool)
+        // sticked at the top of the map.
+        markerHintTip = L.tooltip([0, 0], {
```

Comment:
Not sure if we should use a plain html element, positioned absolute, instead. Then there would be no need to recalculate the position with every move